share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -201,27 +281,92 @@ def mkPostJobRequest ((RunnerInput label cmd vis env dir stdin _res _prefix _ is
     ibytes
     obytes
 
+export def thirdBy (acceptFn: a => Integer): (list: List a) => Triple (one: List a) (two: List a) (three: List a) =
```

Comment:
Meant to comment this early, but `thirdBy` should 
- probably be a stdlib function
- have `acceptFn` be `a => ThirdByGroup` instead of an `Integer`

```ruby
data ThirdByGroup =
    ThirdByGroupFirst 
    ThirdByGroupSecond
    ThirdByGroupThird
```

but I'm not too sure. Leaving it up to reviewers to weigh in